app/models/hearings/sent_hearing_email_event.rb

```diff
@@ -11,6 +11,8 @@ class SentHearingEmailEvent < CaseflowRecord
   belongs_to :sent_by, class_name: "User"
   belongs_to :email_recipient, class_name: "HearingEmailRecipient"
 
+  has_many :sent_hearing_admin_email_events
```

Comment:
```suggestion
  has_one :sent_hearing_admin_email_events
```
At the moment, we never expect to have more than one admin email per sent mail.